src/pages/DatasetCatalog.module.css

```diff
@@ -0,0 +1,111 @@
+.first-child {
+  position: absolute;
+  margin-left: -40px;
+}
+
+.checkbox {
+  margin-top: 0;
+  display: inline;
+}
+
+.checkbox label {
+  padding-top: 1px;
+}
+
+.checkbox[disabled] .regular-checkbox {
+  cursor: not-allowed !important;
+  opacity: 0.65;
+}
+
+.checkbox[disabled] .regular-checkbox:before  {
+  background-color: #eee;
+  border: 1px solid #ccc;
+}
```

Comment:
Where do these original styles that are not in the css index come from?

Reply:
All styles in this module were copied from `index.css`. Not all of them were removed from `index.css`, but the ones defined here will only ever be used by this page. We could use a comprehensive cleanup of `index.css` but the goal of this PR was to unblock your work.